task07/entity/cart.entity.ts

```diff
@@ -0,0 +1,13 @@
+import { ProductEntity } from './product.entity';
+
+export interface CartItemEntity {
+  product: ProductEntity;
+  count: number;
+}
+
+export interface CartEntity {
+  id: string; // uuid
```

Comment:
FYI: Instead of a comment, you can create something like: `type UUID = string;` and assign it to the `id`.